### DISCUSSION TITLE
Get max id from parent

### DISCUSSION THREAD

**Discussion by CROprogrammer** [MarimerLLC/csla#3337] [Questions]
In my BusinessBase (child) object I want to get the max value of IdProperty from all children from parent object (BusinessListBase), but when I try to access parent object in BusinessBase class I always get null parent. Any tips how to solve that issue?

**Comment by kcabral817** [comment]
Can you show the code?

Regards

Kevin
________________________________
From: CROprogrammer ***@***.***>
Sent: Friday, May 12, 2023 6:19:55 AM
To: MarimerLLC/csla ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [MarimerLLC/csla] Get max id from parent (Discussion #3337)


In my BusinessBase (child) object I want to get the max value of IdProperty from all children from parent object (BusinessListBase), but when I try to access parent object in BusinessBase class I always get null parent. Any tips how to solve that issue?

**Reply by rockfordlhotka** [reply]
I don't know if this helps, but `Parent` isn't set until a child has been added to a parent list or object.

Also, in `DynamicListBase` I don't think `Parent` is set, because `DynamicListBase` doesn't really have a parent-child relationship to all the root objects contained in the list.
